### ISSUE TITLE
kubeadm package has 1.7 version

### ISSUE THREAD

**Issue by jbeda** [kubernetes/release#282]
See https://packages.cloud.google.com/apt/dists/kubernetes-xenial-unstable/main/binary-amd64/Packages

```
Package: kubeadm
Version: 1.7.0-alpha.0.1047-7b4bec038c226b-00
```

**Comment by jbeda** [comment]
Is this something that you can take on @mikedanese?  Let me know if I can help out.

cc @luxas

**Comment by luxas** [comment]
@jbeda This is natural, it's the current HEAD

This is what I'll update with the stable deb/rpm v1.6 release

**Comment by mikedanese** [comment]
I agree with @luxas that this is by design. Unstable contains nightly builds.

**Comment by jbeda** [comment]
I'm confused then.  The other packages in the same repo have 1.6 beta versions.

```
Package: kubectl
Version: 1.6.0-beta.3-00
```

I'd expect these to be consistent.

**Comment by mikedanese** [comment]
Hehe, ok. I see why in build.go this is the case but I don't know what the reasoning for making it that way was. I'll let @luxas answer. Which do we want?

**Comment by jbeda** [comment]
Personally, I'd love to have packages that are "prerelease" as we are building a new release.

We kind of need ~3 tracks right?
1 stable
2 unstable -- upcoming releases.  Either major or point releases.  Not sure how to talk about that.
3 head/nightly/ci -- the latest from the CI system

Are we overloading/merging 2 and 3?

**Comment by luxas** [comment]
@jbeda That's exactly how I've implemented it for the debs.

The only problem here is that at the time I wrote this code, I had to make kubeadm unstable the latest CI build, due to that officially released builds weren't available back then IIRC.
That's a thing I'm gonna fix soon, when bumping stable to v1.6

I think we should also consider a 4. legacy/previous/archive channel.

**Comment by luxas** [comment]
Now fixed at HEAD
